currency/templates/currency/currency_form.html

```diff
@@ -6,7 +6,7 @@
 <form action="" method="POST" enctype="multipart/form-data">
     {% csrf_token %}
     {{form}}
-    <input type="submit" value="Add" class="mt16 btn btn-primary left" />
+    <input type="submit" value="Submit" class="mt16 btn btn-primary left" />
```

Comment:
```suggestion
    <input type="submit" value="Update" class="mt16 btn btn-primary left" />
```

Reply:
It is the same template which creates currency, so I've decided to use a more neutral "Submit" and renamed template as currency_form

Reply:
Ok, leave it as it is for now but you probably can detect which action user is doing and rename accordingly while rendering.